reporter/tap.js

```diff
@@ -0,0 +1,77 @@
+(function() {
+
+function reporter( rep ) {
+	if ( rep === "tap" ) {
+		tapReporter();
+	}
+};
+
+reporter.print = function( val ) {
+	console.log( val );
```

Comment:
Should we make this a bit more generic? Like:

```js
console.log.apply(console, arguments);
```

I guess this depends on the circumstances under which `print` will be used.